### PR TITLE
[12.0][MIG] website_sale_product_attribute_filter_visibility: Migration

### PR DESCRIPTION
**Pull request by sergio-teruel**
Hi @cor9003 , can you take a look at the PR?? I want to help you..
If you have any questions, do not hesitate to ask.

### CODE REVIEW

File: website_sale_product_attribute_filter_visibility/tests/test_attribute_filter_visibility.py [hunk 1]
```diff
@@ -5,15 +5,72 @@
...
+                }),
+                (0, 0, {
+                    'attribute_id': self.product_attribute_test_size.id,
+                    'value_ids': [(6, 0, [
+                        self.product_attribute_value_test_size_small.id,
+                        self.product_attribute_value_test_size_large.id])]
+                })]
+        })
+
+        # Active attribute's filter in /shop. By default is disable.
```
Comment (Yajo) [inline website_sale_product_attribute_filter_visibility/tests/test_attribute_filter_visibility.py]:
```suggestion
        # Active attribute's filter in /shop. By default it's disabled.
```

File: website_sale_product_attribute_filter_visibility/tests/test_attribute_filter_visibility.py [hunk 2]
```diff
@@ -5,15 +5,72 @@
...
             "odoo.__DEBUG__.services['web_tour.tour']",
             "website_sale_product_attribute_filter_visibility",
         )
         self.browser_js(
-            url_path="/",
+            url_path="/shop",
             code="%s.run('%s')" % tour,
             ready="%s.tours['%s'].ready" % tour,
-            login="admin" , password="admin",
+            login="admin"
```
Comment (Yajo) [inline website_sale_product_attribute_filter_visibility/tests/test_attribute_filter_visibility.py]:
```suggestion
            login="admin",
```

**Comment by cor9003** [comment]
@sergio-teruel Checked and I see you has move the demo data has to the test and you also have modified the steps working with extra trigger. What is next step ?. Do i Merge it with my PR ?. Sergio ... thanks again ....

**Comment by sergio-teruel** [comment]
Hi @cor9003 Yes, now you merge my proposed PR into your branch. About demo data, In V11, after a odoo commit (I do not remember which) the tours run on a distinct databse cursor so we nedd create data for tests as demo, but in V12 or higher the data por testing pourpose already can now be created when the test is runing.
As strange case, I had to activate the attribute's filter view in the setup method.
About extra_triger, The tour has to find by jquery selector the string text, if it is found, the tour continues to next step.

**Pull request merged**